lib/region/europe/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @module Module for determining if a locale is in the EU.
+ *
+ * Examples:
+ *
+ * // Example 1, loading this module directly:
+ * var europe = require('bv-ui-core/lib/region/europe');
+ * europe.has('en_US'); // false
+ * europe.has('en_GB'); // true
+ *
+ * // Example 2, loading the region module:
+ * var region = require('bv-ui-core/lib/region');
+ * region.has('en_US', 'europe'); // false
+ * region.has('en_GB', 'europe'); // true
+ */
+
+var countryCodes = require('./countryCodes');
+
+module.exports = {
+
+  /**
+   * Determine if a given locale is an EU locale based on the country code.
+   *
+   * @param {String} locale - The locale to be tested. This function accepts
+   *   locales of both xx_XX format and country codes on their own, XX.
+   *
+   * @return {Boolean} Returns true if the locale is an EU locale.
+   *   Returns false otherwise.
+   */
+  has: function (locale) {
+
+    // First, we need to fish out a country code. If the locale is of the xx_XX
+    // format, we need to toss the language part and keep the country code. If
+    // the locale is just a country code already, we can just keep it. Either
+    // way, we can just create split along '_' and grab the last one from the
+    // resulting list.
+    var countryCode = locale.split(/[_\-]/).pop();
```

Comment:
Sorry, I should have elaborated more. A locale of just `fr` normally refers to the language, not the territory, in `fr_FR`. This API isn't intended to be used like `europe.has(language)`, it's about the territory.

That there is no territory included in the input is a sign that the user is misusing the API, and it might be better to alert them that they are doing something wrong instead of giving the technically correct, but wrong answer to the question they think they are asking:

```javascript
europe.has('fr') // false
europe.has('fr_FR') // true
```

The latter is supported, the former is unsupported but still returns a value.